go/vumitools/middleware.py

```diff
@@ -623,6 +623,7 @@ class ConversationMetricsMiddleware(BaseMiddleware):
     SUBMANAGER_PREFIX = "conversation.metrics.middleware"
     RECENT_CONV_KEY = "recent_conversations"
     OLD_RECENT_CONV_KEY = "old_recent_conversations"
+    LOCAL_RECENT_CONVS = set()
```

Comment:
We probably want to create this on each instance (maybe in `.setup_middleware`) so that each instance has its own set.